deps/Makefile

```diff
@@ -485,12 +493,24 @@ LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=clang
 else
 # block default building of Clang
 LLVM_MFLAGS += OPTIONAL_PARALLEL_DIRS=
+ifeq ($(LLVM_VER),svn)	# changed since LLVM 3.9
```

Comment:
Sigh... Here's an easy way: turn around the `ifeq`, and check whether `LLVM_VER` is in 3.3, 3.4, 3.5, 3.6, 3.7 or 3.8 (you can use `subst` to remove the `.0`, `.1` or `.2` minor release numbers).